base/multi.jl

```diff
@@ -1,47 +1,5 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-## multi.jl - multiprocessing
```

Comment:
okay as long as there's a better version somewhere else and this isn't the only place this is listed, seems okay to delete then